### PR TITLE
fix: fix new piece store and spdb

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Fix new piece store and spdb temporarily. This problem will be solved in the future.

### Rationale

N/A

### Example

N/A

### Changes

N/A

### CODE REVIEW

**Review by will-2012 — APPROVED** [review]

File: base/gfspapp/app_options.go [hunk 1]
```diff
@@ -225,7 +226,8 @@ func DefaultGfSpPieceStoreOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) e
 	}
 	pieceStore, err := piecestoreclient.NewStoreClient(&cfg.PieceStore)
 	if err != nil {
-		return err
+		log.Warnw("failed to new piece store", "error", err)
```
Comment (joeylichang) [inline base/gfspapp/app_options.go:229]: "if not use piece store, please ignore: failed to new piece store", "error", err

File: base/gfspapp/app_options.go [hunk 2]
```diff
@@ -197,7 +197,8 @@ func DefaultGfSpDBOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) error {
 	dbCfg := &cfg.SpDB
 	db, err := sqldb.NewSpDB(dbCfg)
 	if err != nil {
-		return err
+		log.Warnw("failed to new sp db", "error", err)
```
Comment (joeylichang) [inline base/gfspapp/app_options.go:200]: ditto.

**Review by joeylichang — APPROVED** [review]

File: base/gfspapp/app_options.go (hunk 1 above)
Comment (sysvm) [reply base/gfspapp/app_options.go]: fixed

File: base/gfspapp/app_options.go (hunk 2 above)
Comment (sysvm) [reply base/gfspapp/app_options.go]: fixed

**Review by joeylichang — APPROVED** [review]

**Pull request merged**